frontend/src/metabase/query_builder/components/dataref/DatabaseSchemasPane.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint "react/prop-types": "warn" */
+import React, { useMemo } from "react";
+import { ngettext, msgid } from "ttag";
+
+import Search from "metabase/entities/search";
+import type { Card } from "metabase-types/api";
+import type { State } from "metabase-types/store";
+import Database from "metabase-lib/lib/metadata/Database";
+import {
+  NodeListItemLink,
+  NodeListItemName,
+  NodeListItemIcon,
+  NodeListTitle,
+  NodeListContainer,
+  NodeListIcon,
+  NodeListTitleText,
+  ModelId,
+} from "./NodeList.styled";
+
+interface DatabaseSchemasPaneProps {
+  show: (type: string, item: unknown) => void;
```

Comment:
It's preferred to use `on*` naming for callbacks like that. In this case, it can be e.g. `onItemClick` or something

Reply:
Same comment for the `SchemaPane` component

Reply:
Good point. I've carefully renamed it in all the Data Reference panels. https://github.com/metabase/metabase/pull/25773/commits/724b3c356a17710f57cd9c83e0e596d4d65f5f6a